akka-http-core/src/main/scala/akka/http/scaladsl/Http.scala

```diff
@@ -258,7 +258,8 @@ class HttpExt(private val config: Config)(implicit val system: ActorSystem) exte
   def clientLayer(hostHeader: Host,
                   settings: ClientConnectionSettings,
                   log: LoggingAdapter = system.log): ClientLayer =
-    OutgoingConnectionBlueprint(hostHeader, settings, log)
+    RedirectSupportStage(settings.redirectSettings).atop(OutgoingConnectionBlueprint(hostHeader, settings, log))
```

Comment:
I can think of some more aspects that'll come up (like caching or proxying). Should they all be baked into the main blueprint?